modules/luci-base/htdocs/luci-static/resources/luci.js

```diff
@@ -1192,6 +1192,8 @@
 			if (env.base_url == null)
 				this.error('InternalError', 'Cannot find url of luci.js');
 
+			env.cgi_base = env.scriptname.substring(0, env.scriptname.lastIndexOf('luci'));
```

Comment:
I think you better should use `L.env.scriptname.replace(/\/[^\/]+$/, '')` here, to take the current program path minus the last path segment. This should result in `/cgi-bin` (minus the trailing slash)